### DISCUSSION TITLE
script setup executed twice?

### DISCUSSION THREAD

**Discussion by NeonMika** [nuxt/nuxt#16281] [Questions] [answered]
I just use exampel code from the documentation with one `console.log()` added.
The output of this `console.log()` is visible once in the browser's dev tab, but twice on the console (with two different numbers being printed).
Why do I see two outputs on the console (running `yarn dev`)?

```
<template>
  <div>
    {{counter}}
  </div>
</template>

<script setup>
const counter = useState('counter', () => Math.round(Math.random() * 1000))
console.log(counter.value) // This prints twice on the terminal console, counter on the website shows the first value.
</script>
```

**Comment by danielroe** [comment, marked as answer]
Likely because your browser is requesting `/favicon.ico` and (as you don't have one) this is being handed with Nitro, which renders it using `app.vue`.

**Reply by elHornair** [reply]
I just run into this too. Would it be a good idea to deliver a default favicon with Nuxt?

**Reply by psycongaroo** [reply]
@elHornair that's a great idea

**Reply by NeonMika** [reply]
I support that!
I was so confused when that happend, and a simple default favicon would have saved me from going into "full error cause reseach mode".

**Reply by MuhammadM1998** [reply]
This happened to me and I solved it by using a favicon but I don't really grasp the cause, could any one elaborate on 'being handed with Nitro, which renders it using `app.vue`' ?
